dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -475,7 +475,9 @@ public boolean checkUserExists(@ApiParam(hidden = true) @Parameter(hidden = true
     @ApiOperation(value = "Get information about tokens with user id.", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, response = Token.class, responseContainer = "List")
     public List<Token> getUserTokens(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User user,
             @ApiParam("User to return") @PathParam("userId") long userId) {
-        checkUserId(user, userId);
+        if (!user.isCurator() && !user.getIsAdmin()) {
```

Comment:
So, currently the code is using `this.tokenQuery.tokens$` but that is wrong since it accesses the logged in user's tokens. Now with this change, we will be able to use `this.usersService.getUserTokens(this.user.id)` to get the linked accounts for the desired user.

Reply:
Oh I see, so there's a UI change that will be needed following this change. Makes sense